plugins/module_utils/common/json_to_xml.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+"""
+The index_of plugin common code
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import xmltodict
```

Comment:
This should be a conditional import based on conversion engine value passed to filter plugin by user. Default to `xmltodict`